ReactWindows/ReactNative/UIManager/LayoutShadowNode.cs

```diff
@@ -165,7 +168,7 @@ public void SetMargins(int index, float margin)
             ViewProps.PaddingTop,
             ViewProps.PaddingBottom,
             DefaultSingle = Undefined)]
-        public void SetPaddings(int index, float padding)
+        public virtual void SetPaddings(int index, float padding)
```

Comment:
Didn't we avoid making this virtual for TextInput?  I believe we somehow just piggy-backed off the underlying Padding value that was set on CSSNode.